src/main/java/bisq/core/app/AppSetupWithP2PAndDAO.java

```diff
@@ -18,7 +18,7 @@
 package bisq.core.app;
 
 import bisq.core.dao.DaoManager;
-import bisq.core.dao.proposal.ProposalCollectionsManager;
+import bisq.core.dao.proposal.ProposalCollectionsService;
```

Comment:
Actually following DDD it would be rather a repository. I just found in a past job the repository naming somehow artificial if you don't know that it is a DDD term, so I never got too convinced about that term. I used service as a more general term, it serves the clients (usually UI views or other services) with domain data and logic.